_delphi_utils_python/delphi_utils/validator/static.py

```diff
@@ -96,7 +96,9 @@ def check_missing_date_files(self, daily_filenames, report):
 
         # Diff expected and observed dates.
         expected_dates = self.params.time_window.date_seq
-        max_expected_lag_overall = max(self.params.max_expected_lag.values())
+        max_expected_lag_overall = 10 if self.params.max_expected_lag else 
+            max(self.params.max_expected_lag.values())
```

Comment:
Looks like the logic here is flipped (or you need a `not`)

Reply:
Yeah i confused myself, should be fixed now